src/Learning/KWUserInterface/KWDatabaseTransferView.cpp

```diff
@@ -191,6 +201,10 @@ void KWDatabaseTransferView::Open()
 		}
 	}
 
+	// Les dictionnaires non stockables sont ranges en derniers, apres une ligne blanche
+	if (sNonStorableClassNames.GetLength() > 0)
+		sClassNames += "\n\n" + sNonStorableClassNames;
```

Comment:
IMHO, from an ergonomic standpoint, one might consider e.g. `\n--------\n` instead of `\n\n`? Thus, the storable / non-storable separation would be even mode visible?

Reply:
J'avais fait ce type de test, qui est effectivement plus sympa dans la liste d'aide à la saisie.
Mais cela pose plusieurs problème:
- "--------" serait alors considéré comme un nom possible de dictionnaire
  - mais si on le choisit, on aura un message de type "Dictionnary -------- not found"
  - ou alors, il faut développer un code spécifique partout ou c'est nécessaire pour éviter ce comportement non désrié
- de plus, on peut en théorie avoir un dictionnaire valide dont le nom est précisément "--------"

L'idéal sera d'avoir un séparateur uniquement de type ergonomie, mais que l'on ne peut pas sélectionner. Ou d'avoir les dictionnaire non stockable en grisé.
Mais cela demande d'étendre le moteur d'interface implémenté dans Norm (cf. NormGUI), uniquement pour ce cas qui ne devrait se produire que de façon très marginale.

Le rapport coût-bénéfice de la solution actuelle a atteint une forme d'optimalité.

Reply:
Ok, got it.